### ISSUE TITLE
Not updating realtime 

### ISSUE THREAD

**Issue by umar28dec** [olayinkaos/pusher-laravel-realtime-table#2]
follow the installation process properly, app is working fine but not updating real time guide me.

**Comment by haleyngonadi** [comment]
Same thing with me ):

**Comment by olayinkaos** [comment]
Hi @umar28dec @haleyngonadi. Are you getting any errors in Console?

**Comment by olayinkaos** [comment]
Also, please check that the correct Pusher settings are in your `.env`. You can also debug with the Pusher console to see if it's actually triggering any events.

**Comment by haleyngonadi** [comment]
@olayinkaos When I arrive on the page with the listener, Pusher logs an event.. but when I click the actual card to log a new score, the event isn't triggered!

**Comment by olayinkaos** [comment]
Oops. Just realized there's a step missing in the README. You need to run `npm run dev` to compile the JS assets so the table can update in real time! Can you try doing that and let me know?

Following through with the tutorial [here](https://pusher.com/tutorials/realtime-table-laravel/) also really helps understand what's going on.

**Comment by haleyngonadi** [comment]
@olayinkaos I actually did have npm run watch running when I changed the vue files! Also tried npm run dev, no luck ): 

I think there's some kind of disconnect between Laravel's event trigger and the echo listener because I get a response that the score was updated but nothing on pusher!

**Comment by haleyngonadi** [comment]
I figured this out – so silly!

`php artisan queue:listen
`

**Comment by umar28dec** [comment]
Yes, I follow the complete process including npm things and not getting any error in console

**Comment by umar28dec** [comment]
it's updated on pusher debug console as well

**Comment by umar28dec** [comment]
I am on local server, is it needed live server?

**Comment by adeguntoro** [comment]
Same here, i follow all comment, still no luck. For plain PHP i can get notification, but for laravel ? nope.

**Comment by webcreative24** [comment]
same issue. I don't have any error in console but the data doesn't updates in real time
